docs/Project.toml

```diff
@@ -1,3 +1,6 @@
 [deps]
 Documenter = "e30172f5-a6a5-5a46-863b-614d45cd2de4"
 FluorophoreColors = "d4071afc-4203-49ee-90bc-13ebeb18d604"
+
+[compat]
+Documenter = "0.27"
```

Comment:
If we don't need fancy features, then it's perhaps a good idea to just drop the compat. I'm okay either way.

Reply:
Mostly it's to avoid breakage if Documenter 0.28 or higher breaks some things we use, but that hasn't happened in quite some time so...:shrug:

I'll change it.